projects/1.16/assets/vanilla-cookbook/vanillacookbook/lang/zh_cn.json

```
@@ -1 +1,123 @@
+	"item.vanillacookbook.trail_mix": "干果",
+	"item.vanillacookbook.hot_chicken_wing": "熟鸡翅",
+	"item.vanillacookbook.fish_and_chips": "炸鱼薯条",
+	"item.vanillacookbook.apple_soda": "苹果汽水",
+	"item.vanillacookbook.berry_soda": "甜浆果汽水",
+	"item.vanillacookbook.cactus_soda": "仙人掌汽水",
+	"item.vanillacookbook.melon_soda": "西瓜汽水",
+	"item.vanillacookbook.berry_candy": "甜浆果糖果",
+	"item.vanillacookbook.meat_stew": "炖肉煲",
+	"item.vanillacookbook.meatloaf": "肉饼",
+	"item.vanillacookbook.meatball": "肉丸",
+	"item.vanillacookbook.spicy_meatball": "辛辣肉丸",
+	"item.vanillacookbook.kebab": "烤肉串",
+	"item.vanillacookbook.chorus_juice": "紫颂果汁",
+	"item.vanillacookbook.chorus_soda": "紫颂果汽水",
+	"item.vanillacookbook.french_toast": "法式吐司",
+	"item.vanillacookbook.magma_jelly": "岩浆果冻",
+	"item.vanillacookbook.wart_pie": "疣派",
+	"item.vanillacookbook.baked_apple": "烤苹果",
+	"item.vanillacookbook.cooked_sea_pickle": "熟海泡菜",
+	"item.vanillacookbook.cooked_turtle_egg": "熟海龟蛋",
+	"item.vanillacookbook.sea_pickle_juice": "海泡菜汁",
+	"item.vanillacookbook.apple_sauce": "苹果酱",
+	"item.vanillacookbook.potato_pancake": "土豆煎饼",
+	"item.vanillacookbook.sword_lollipop": "棒棒糖剑",
+	"item.vanillacookbook.sea_pickle_soda": "海泡菜汽水",
+	"item.vanillacookbook.apple_chips": "苹果脆片",
+	"item.vanillacookbook.fungus_stew": "下界菌煲",
+	"item.vanillacookbook.root_salad": "下界菌索沙拉",
+	"item.vanillacookbook.netherite_apple": "下界合金苹果",
+	"item.vanillacookbook.berry_milk": "甜浆果奶",
+	"item.vanillacookbook.stuffed_sea_pickle": "填馅海泡菜",
+	"item.vanillacookbook.mushroom_scrambled_eggs": "蘑菇炒蛋",
+	"item.vanillacookbook.pumpsicle": "南瓜棒",
+	"item.vanillacookbook.pumpkin_cookie": "南瓜饼干",
+	"item.vanillacookbook.chocolate_egg": "巧克力蛋",
+	"item.vanillacookbook.sunflower_seed_cookie": "向日葵籽饼干",
+	"item.vanillacookbook.cactus_slice": "仙人掌片",
+	"item.vanillacookbook.cooked_cactus_slice": "熟仙人掌片",
+	"item.vanillacookbook.pizza": "披萨",
+	"item.vanillacookbook.berry_cake_slice": "甜浆果蛋糕切片",
+	"item.vanillacookbook.book_cake_slice": "蛋糕书切片",
+	"item.vanillacookbook.carrot_cake_slice": "胡萝卜蛋糕切片",
+	"item.vanillacookbook.cheesecake_slice": "芝士蛋糕切片",
+	"item.vanillacookbook.chocolate_cake_slice": "巧克力蛋糕切片",
+	"item.vanillacookbook.chorus_cake_slice": "紫颂果蛋糕切片",
+	"item.vanillacookbook.ice_cream_cake_slice": "冰淇淋蛋糕切片",
+	"item.vanillacookbook.mohnkuchen_slice": "虞美人籽蛋糕切片",
+	"item.vanillacookbook.rose_cake_slice": "玫瑰蛋糕切片",
+	"block.vanillacookbook.berry_cake": "甜浆果蛋糕",
+	"block.vanillacookbook.carrot_cake": "胡萝卜蛋糕",
+	"block.vanillacookbook.cheesecake": "芝士蛋糕",
+	"block.vanillacookbook.chocolate_cake": "巧克力蛋糕",
+	"block.vanillacookbook.mohnkuchen": "虞美人籽蛋糕",
+	"block.vanillacookbook.ice_cream_cake": "冰淇淋蛋糕",
+	"block.vanillacookbook.rose_cake": "玫瑰蛋糕",
+	"block.vanillacookbook.chorus_cake": "紫颂果蛋糕",
+	"block.vanillacookbook.book_cake": "蛋糕书",
+	"block.vanillacookbook.pancake_stack": "煎饼堆",
+	"block.vanillacookbook.brownie_tray": "布朗尼块 ",
+	"advancements.more_balanced_diet.title": "一个更均衡的饮食",
```

显然 neta 原版进度名称 A Balanced Diet → 均衡饮食。没有“一个”
```suggestion
	"advancements.more_balanced_diet.title": "更均衡的饮食",
```
后面的描述建议和原版搭点边。